svid/jwtsvid/tests/check_svid.c

```diff
@@ -57,7 +57,7 @@ START_TEST(test_jwtsvid_ParseInsecure)
     // jwtbundle_Bundle *bundle = jwtbundle_Load(td,
     // "./resources/jwk_keys.json", &err);
 
-    char token[]
+    char token1[]
```

Comment:
For each declared token, I'd use a more meaningful variable name, like token_with_audience_x or invalid_token, etc.